.github/labeler/contents.yml

```diff
@@ -11,4 +11,4 @@
 
 "Scope: Tests Only":
   - all:
-    - "+(factories|features|spec|test|.github)/**/*"
+    - "{factories,features,spec,test,.github}/**/*"
```

Comment:
When minimatch is converting a glob to a regex, it checks whether the pattern starts with a `.`. If it doesn't start with a dot, then it [adds a negative lookahead to the regex](https://github.com/isaacs/minimatch/blob/6410ef32f59e4842121ca13eefacdf0b3da8533c/minimatch.js#L440-L445) to ensure that the pattern won't match to anything that starts with a dot. This prevents the `+(factories|...|.github)` pattern from matching to anything in the `.github` directory, because the pattern doesn't start with a dot but the `.github` directory does.

Braces, however, are processed before the pattern is turned into a regex. Specifically, the braces above turn the pattern into five separate patterns:
- `factories/**/*`
- `features/**/*`
- `spec/**/*`
- `test/**/*`
- `.github/**/*`

Then each of those patterns is turned into a separate regex, and if any one of those regexes matches, it counts as matching the glob. Because the `.github/**/*` pattern *does* start with a `.`, it doesn't have the special negative lookahead to prevent it from matching dotfiles, so it will happily match to the `.github` directory.